scripts/install.sh

```diff
@@ -723,7 +730,7 @@ if [ $DIRECTORY != "-" ]; then
   fi;
 fi;
 
-echo "Installing OctoDash ..."
+echo "Installing OctoDash "${version[7]}, $arch..."
```

Comment:
```suggestion
echo "Installing OctoDash ..."
```

Having the same output line twice might be confusing. After the version and arch has been printed they shouldn't be printed again IMHO.

Reply:
The only reason I did this, is due to the first version print is scrolled due to package deps. It is not needed, just thought it would be nice to have it shown for each, so you don't have to scroll to find it.

Reply:
Yeah makes sense. I'd still remove the last one then, are you fine with that? Are the `...` printed correctly though? Shouldn't there be another `"` right in front of them?

Reply:
Yea, looks like I missed that. It has been fixed, with the original space that was there. The other line has been reverted also.

Reply:
Thanks! 👍